src/instrumentations/xhr.ts

```diff
@@ -742,6 +761,14 @@ export function instrumentXHR(config: XhrConfig) {
           taskCounter.increment();
           xhrMem.sendStartTime = hrTime();
           currentSpan.addEvent(EventNames.METHOD_SEND);
+          if (config.networkBodyCapture) {
+            currentSpan.setAttribute(
+              'http.request.body',
+              typeof requestBody === 'string'
+                ? requestBody
+                : JSON.stringify(requestBody)
```

Comment:
better to handle the case when stringify throws. also I wonder if we want to set a cap on body size ?

Reply:
added!